sunpy/net/jsoc/jsoc.py

```diff
@@ -582,7 +582,7 @@ def get_request(self, requests, path=None, overwrite=False, progress=True,
         dl_set = True
         if not downloader:
```

Comment:
So the issue here is that this if statement will not be hit from `Fido.fetch` as we pass in a `Downloader` instance.

Reply:
`max_conn` is an instance variable.
My OOPS Instructor would shake his head in disapproval if I modify an instance variable without getters and setters after an object has been created. Should we implement that as a property in parfive first?

Reply:
I am not sure it's worth it. Interface control is great and all, but what's a property in Python doing if not just changing an instance variable underneath :grinning: 

The other thing we could do here is just disable multipart altogether for JSOC?

Reply:
Which one would you prefer? 
I for one think that we should change the instance variable here for now and implement checks to see if the values passed by the users exceed 8 connections. Raise a warning and reduce the connections to our default values.

Reply:
Sounds good but let's warn on >10 and default to 8. (Given 10 is the max allowed).

Reply:
If the limit is 10, do we want to be that close to it by default?

Reply:
Well you would have to be running two separate python processes or other downloads from JSOC to go over it. We want to get people their data as quickly as we can imo. We should obviously document this so if it goes wrong people have an idea why.